warehouse/query-core/src/main/java/datawave/query/config/LookupUUIDTune.java

```diff
@@ -54,6 +54,8 @@ public void configure(BaseQueryLogic<Entry<Key,Value>> logic) {
             rsq.setSpeculativeScanning(speculativeScanning);
             rsq.setCacheModel(enableCaching);
             rsq.setPrimaryToSecondaryFieldMap(primaryToSecondaryFieldMap);
+            if (enforceUniqueTermsWithinExpressions)
+                rsq.setEnforceUniqueTermsWithinExpressions(true);
```

Comment:
Is the conditional here really necessary?

Reply:
Nope, resolving in next commit.